packages/web-app-search/src/portals/SearchBar.vue

```diff
@@ -179,7 +179,7 @@ export default defineComponent({
     }
   },
   created() {
-    this.debouncedSearch = debounce(this.search, 200)
+    this.debouncedSearch = debounce(this.search, 500)
```

Comment:
Why did we increase this again?

Reply:
the backend is not able to handle the load atm, we need it in experimental and i expect its the same for master.